fix(agnostic): prefer manager built-in version checker and suggest `arm-v8a` arch (#15)
SodaWithoutSparkles/revanced-troubleshooting-guide: step-by-step-general/03-get-files.md

Do **NOT** try to find the manager version shown in the image, it might be outdated.
!!!
===
2. In patcher tab, tap Select an Application
3. Find the app that you want to patch.
!!! If you cannot find the app...
If you have not installed the app on your phone, the package name would be shown instead.
These would be indicated by a missing icon. You can find a lookup table [here](/05-versions.md).
You can also find the package name for an app by searching "\<app name\> package name" in a browser.

If you are certain that you have the app installed, using the search bar may help.
!!!

> 5. The suggested version of the app would be shown. 
> !!! If it said "Any version"

Reviewer: ```suggestion
5. The suggested version of the app will be shown. 
!!! If it says "Any version"
```

This means that nobody has reported a version that did not work *yet*.
It means that you can *try* the latest version of apk available.
In case it doesn't work, go back a few versions and check if it works, and repeat if it doesn't.
You can also report back the version that works for you at [Discord](https://discord.gg/revanced) or [Github (if nobody else has reported)](https://github.com/ReVanced/revanced-patches/issues)
!!!

> 5. You can tap the version shown to open an online search result of the suggested apk. You can also just google it.

Reviewer: Maybe recommend the search result from apkmirror.com

Author: good idea

> 6. You can tap the version shown to open an online search result of the suggested apk. APKMirror usually has them. You can also just google it.

Reviewer: ```suggestion
6. You can tap the version shown to open an online search result of the suggested apk. Or you can google it. APKMirror will usually have the apk.
```

> 6. If there are multiple variants, you most likely need the one marked `arm64-v8a` and `APK` (not bundle).

Reviewer: Combining the arch information with the file type can cause confusion. Separating them may be simpler:

```suggestion
6. If there are multiple variants, choose the one marked `APK` (not bundle) with the "arch" marked as universal. If no universal arch is available, you most likely need the `arm64-v8a` or `armeabi-v7a` one.
```

=== Image for reference

> ![This variant is both `arm-v8a` and is an apk](https://github.com/SodaWithoutSparkles/ReVanced-troubleshooting-guide/blob/main/screenshots/022-verify-apk-correct-arch?raw=true)

Reviewer: The screenshot doesn't highlight that this one is an apk and not a bundle

Author: I thought I have drawn a blue box over it... 
maybe I will draw one more arrow

===

> 8. Verify that it said "DOWNLOAD APK" not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.

Reviewer: ```suggestion
8. Verify that it says "DOWNLOAD APK" and not "DOWNLOAD APK BUNDLE". The file should end with `.apk` not `.apks` or `.apkm`.
```
